### PR TITLE
Bump to 4.6.0

### PR DESCRIPTION
**Pull request by AnasNaouchi**
Bump to 4.6.0

### CODE REVIEW

**Review by aashishgurung — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # CHANGE LOG
+## v4.6.0
+
+* **NEW** Added `duitnow_obw sourceType to enable charge creation for duitnow` source types (#156)
```
Comment (nuxzero) [inline CHANGELOG.md:4]: I think the description should not be in the code highlight. Can you rephrase this?

Comment (AnasNaouchi) [reply CHANGELOG.md]: updated @nuxzero

**Review by nuxzero — APPROVED** [review]
